### PR TITLE
chore(browser): add browser ui scripts to run root browser tests

### PR DESCRIPTION
**Pull request by userquin**
### Description

This PR simplifies run the browser ui tests adding 2 scripts (for playwright and webdriverio providers)

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. If the feature is substantial or introduces breaking changes without a discussion, PR might be closed.
- [ ] Ideally, include a test that fails without this PR but passes with it.
- [ ] Please, don't make changes to `pnpm-lock.yaml` unless you introduce a new test example.

### Tests
- [ ] Run the tests with `pnpm test:ci`.

### Documentation
- [ ] If you introduce new functionality, document it. You can run documentation with `pnpm run docs` command.

### Changesets
- [ ] Changes in changelog are generated from PR name. Please, make sure that it explains your changes in an understandable manner. Please, prefix changeset messages with `feat:`, `fix:`, `perf:`, `docs:`, or `chore:`.

### CODE REVIEW

File: test/browser/package.json [hunk 1]
```diff
@@ -10,7 +10,9 @@
     "test:safaridriver": "PROVIDER=webdriverio BROWSER=safari pnpm run test:unit",
     "test-fixtures": "vitest",
     "test-mocking": "vitest --root ./fixtures/mocking",
-    "coverage": "vitest --coverage.enabled --coverage.provider=istanbul --browser.headless=yes"
+    "coverage": "vitest --coverage.enabled --coverage.provider=istanbul --browser.headless=yes",
+    "test:browser:playwright": "PROVIDER=playwright vitest --ui",
```
Comment (sheremet-va) [inline test/browser/package.json:14]: 1. there is already test-fixtures, adding PROVIDER= is not that longer
2. why --ui? it's already built in

**Review by sheremet-va — CHANGES_REQUESTED** [review]

File: test/browser/package.json (hunk 1 above)
Comment (userquin) [reply test/browser/package.json:14]: I haven't playwright installed, I use webdriverio :sweat_smile: :

imagen

Comment (userquin) [reply test/browser/package.json:14]: This way we can run the script

Comment (userquin) [reply test/browser/package.json:14]: removing the `--ui` flag

Comment (sheremet-va) [reply test/browser/package.json:14]: ah, you are saying that on windows it's harder to set env

Comment (userquin) [reply test/browser/package.json:14]: yes, I neet to use `set PROVIDER=xxxx`

**Review by sheremet-va — APPROVED** [review]

**Pull request merged**